scipy/special/orthogonal.py

```diff
@@ -1041,8 +1193,34 @@ def he_roots(n, mu=False):
 
 
 def hermitenorm(n, monic=False):
-    """Return the nth order normalized Hermite polynomial, He_n(x), orthogonal
-    over (-inf,inf) with weighting function exp(-(x/2)**2)
+    r"""Normalized (probabilist's) Hermite polynomial.
+
+    Defined by
+
+    .. math::
+
+        He_n(x) = (-1)^ne^{x^2/2}\frac{d^n}{dx^n}e^{-x^2/2};
+
+    :math:`He_n` is a polynomial of degree :math:`n`.
+
+    Parameters
+    ----------
+    n : int
+        Degree of the polynomial.
+    monic : bool, optional
+        If `True`, scale the leading coefficient to be 1. Default is
+        `False`.
+
+    Returns
+    -------
+    He : orthopoly1d
+        Hermite polynomial.
+
+    Notes
+    -----
+    The polynomials `He_n` are orthogonal over :math:`(-\infty,
+    \infty)` with weight function :math:`e^{-(x/2)^2}`.
```

Comment:
Is the weight function correct as written -- is it (x/2)**2 or x**2/2 ? Ditto in the docstring of he_roots.